spec/features/admin/order_shipments_links_to_returns_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.feature 'Return Button Under Order#edit', type: :feature do
+  let!(:order) { create :shipped_order }
+  stub_authorization!
+
+  before :each do
+    visit spree.edit_admin_order_path(order)
+  end
+
+  scenario 'displays the button' do
```

Comment:
Feature specs allow this feature/scenario DSL which is a direct alias, just like context to describe and it. The idea being that scenario/feature reads more like acceptance or customer based tests.